pandas/core/categorical.py

```diff
@@ -127,6 +127,8 @@ def maybe_to_categorical(array):
     """ coerce to a categorical if a series is given """
     if isinstance(array, (ABCSeries, ABCCategoricalIndex)):
         return array._values
+    elif isinstance(array, np.ndarray):
```

Comment:
how is this hit?

Reply:
can you add a comment to the doc-string that this is *only* an internal method.

Reply:
May I prefix it with an underscore? It's used in a single place only.